### PR TITLE
Add Apache Geode Support

### PR DESCRIPTION
**Pull request by tzolov**
- Introduce Vagrant private network at 192.168.68.8 to allow Geode clients access the cluster members
- Add GFSH support
- Fix Cassandra network IP issue

### CODE REVIEW

**Comment by michaelmior** [comment]
I haven't tested this, but did a quick skim and LGTM. I'll try to run other ITs later this week to make sure they still work.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Note: it might take 10-30 minutes depending on your machine and internet connect
 
 # List of created databases
 
+* Geode (port 10334)
```
Comment (julianhyde) [inline README.md]: Can you change this to "Apache Geode".

Also please change "Cassandra" to "Apache Cassandra" (not sure when that mistake crept in!)

Comment (michaelmior) [reply README.md]: Good call! I'm guessing the Cassandra one was me :(

Comment (tzolov) [reply README.md]: Sure i will. Actually i though about it but then deiced to follow the `Cassandra` example :)
I will fix both

**Comment by tzolov** [comment]
I've also removed a bunch of unused domain classes. All test dataset are loaded from JSON docs and are converted on the fly into [PdxInstances](https://geode.apache.org/docs/guide/13/developing/data_serialization/program_application_for_pdx.html).
In the future i may add Java class based schema definitions but until then those domain classes are redundant.

**Pull request merged**